doc/modules/classification_threshold.rst

```diff
@@ -146,6 +146,13 @@ also possible to manually set the decision threshold using the class
 :class:`~sklearn.model_selection.FixedThresholdClassifier`. In case that you don't want
 to refit the model when calling `fit`, you can set the parameter `prefit=True`.
 
+.. note::
+  Given a calibrated classifier and a dataset is representative and large enough,
```

Comment:
I think the calibration is enough, whatever the size of the dataset. But of course training a calibrated classifier and assessing its calibration requires a large enough datasets.